src/librustc_typeck/check/wfcheck.rs

```diff
@@ -209,6 +210,10 @@ fn check_associated_item<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
                     fcx.register_wf_obligation(ty, span, code.clone());
                 }
             }
+            ty::AssociatedKind::Existential => {
```

Comment:
Hmm. I... don't think this concept even makes sense, does it? I would rather expect that a `existential type` in an impl is more like `-> impl Trait` in a function -- it desugars to the pair of an existential type that is scoped to the impl and defined within. On the **trait** side, this is just a plain associated type.

Reply:
yea, I was just following the RFC, but all of the associated existential code is stubbed out anyway. I'll remove it from this PR entirely

Reply:
yes; my point is that even if we allow the syntax I wouldn't necessarily expect it to show up at the `ty` level (though perhaps it will wind up being necessary for...reasons)